src/opcodes/opcodes.cairo

```diff
@@ -174,6 +194,27 @@ pub mod Opcode {
         engine.dstack.push_int(a + b);
     }
 
+    fn opcode_greaterthan(ref engine: Engine) {
+        let a = engine.dstack.pop_int();
+        let b = engine.dstack.pop_int();
+        engine.dstack.push_int(if a > b {
+            0
+        } else {
+            1
+        });
+    }
+
+    fn opcode_within(ref engine: Engine) {
+        let a = engine.dstack.pop_int();
```

Comment:
Would be prefered to make the variable naming here more explicit, like min_value, max_value, ...

Reply:
Will rename the variables now

Reply:
Hi I just pushed an update to rename the variables to be more explicit